mesonbuild/interpreter.py

```diff
@@ -2311,7 +2311,10 @@ def func_find_program(self, node, args, kwargs):
         return progobj
 
     def func_find_library(self, node, args, kwargs):
-        raise InvalidCode('find_library() is removed, use the corresponding method in a compiler object instead.')
+        raise InvalidCode("find_library() is removed, use the corresponding method in a compiler object instead.\n"
```

Comment:
`'` quotes are used in this file, double-quotes are inconsistent.

Reply:
Also instead of corresponding method you could write `meson.get_compiler('name').find_library()`.

Reply:
And if you do then double quotes are justified because you would embed single quotes in them and this is a [preferred style](https://www.python.org/dev/peps/pep-0008/#string-quotes).